### DISCUSSION TITLE
README needs an example video

### DISCUSSION THREAD

**Discussion by lsig-ks** [zk-org/zk-nvim#180] [General]
Hi, I am very interested in learning about zk and the zk-neovim plugin, but I am not completely sure what it does and how it works. It would be great if you could add videos to the README so I can get a better idea on the setup and workflow of the plugin and the tool in general. The zk repo has a video but I am most interested in working with the tool through neovim. 

Thanks for the CLI and the Neovim plugin!

**Comment by tjex** [comment]
Heyo. I totally understand.
There is a bit of friction to get going with zk and zk-nvim.

While a video would be great, and I have thought about it, I don't personally 
have the time to put one together.

There have been initial discussions and planning about putting together a better 
website for just this purpose. Which carries more value per measure of 
maintainer work hours than videos. As when things change, updating a few lines 
in the documentation is much quicker than recording another video.

So an updated website is the plan of action for now.

Try work your way from: 
https://github.com/zk-org/zk/blob/main/docs/getting-started.md

If you get really stuck and can't find any relevant discussion topics here, feel 
free to make a new discussion with the problems you're facing. 

Give it a thoughtful title too, so others can find it in the future :)

**Comment by mickael-menu** [comment]
Shivan made a great video a few years ago. The basics should be the same, but there are a few more Neovim commands available now.

[![](https://img.youtube.com/vi/UzhZb7e4l4Y/0.jpg)](https://www.youtube.com/watch?v=UzhZb7e4l4Y)

**Reply by lsig-ks** [reply]
Thanks!

**Reply by tjex** [reply]
Ah rad. Had no idea. Will put this in the readme.
